source/content/supported-drupal.md

```diff
@@ -17,19 +17,18 @@ The following table indicates availability of the specified Drupal version, as w
 
 | Drupal Version | Available | Recommended | Supported |
 | ----------- | :---------: | :---------: | :---------: |
-| 11          | ⚠️ <Popover title="Drupal 11 Availability" content="See the <a href='#drupal-11-on-pantheon'>following section</a> for how to set up Drupal 11 on Pantheon today." />        | <span style="color:green">✔</span>           | <span style="color:green">✔</span>
+| 11          | <span style="color:green">✔</span> | <span style="color:green">✔</span>           | <span style="color:green">✔</span>
 | 10          | <span style="color:green">✔</span>         | <span style="color:green">✔</span>           | <span style="color:green">✔</span>          |
 | 9           | <span style="color:green">✔</span>          | <span style="color:green">✔</span>           | <span style="color:green">✔</span>         |
```

Comment:
Drupal 9 has been EOL for a while, it could be downgraded to supported, but not recommended.

Reply:
@greg-1-anderson ah, thanks that makes sense. I'll update recommended for D9 from ✅  to ❌ 

Since D9 is no longer an option during site creation in the dashboard, should we also update availability for D9 from ✅  to ❌  ? if there's another way to spin up a new D9 site (say to migrate a live D9 site to Pantheon from another platform), do we want to instead use :warning: with instructions in a tooltip or subsection below the table? are the steps effectively to spin up a d10 site and downgrade to D9 manually via composer?

Reply:
Yes, you can start up a new D10 site and downgrade, or you can create a new D9 site from Terminus, or import an existing D9 site. Also, we have a lot of existing D9 sites on the platform, so I'd keep d8 and d9 as "supported".

Reply:
What is the terminus command to spin up a new D9 site?

Reply:
@stevector @greg-1-anderson I pushed a new commit to address this feedback, can you review again and let me know what you think?

Reply:
reviewed w/greg in slack thread and we opted to swap ⚠️ for ✔️ w/tooltips for d8/9 availability